### PR TITLE
Front-end tech stack: Start off with server-side rendering

### PR DESCRIPTION
**Pull request by alexsoble**
## What does this change? 

Adds ADR for front-end tech stack.

## What does this change look like?

[Preview link.](https://github.com/18F/OPRE-Unicorn/blob/ars/25/start-off-with-server-side-rendering.md/docs/adr/006-start-off-with-server-side-rendering.md) 👁️ 

## GitHub cards

This should complete #25!

### CODE REVIEW

**Review by carjug — APPROVED** [review]
Thanks for this, Alex! LGTM :shipit:

**Review by ninamak — APPROVED** [review]
LGTM!

**Comment by alexsoble** [comment]
@amymok If you approve, could you also merge on approving? Thanks!

File: docs/adr/006-start-off-with-server-side-rendering.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# 6. Front-end tech stack: Start off with server-side rendering
+
+Date: 2021-07-14
+
+## Status
+
+Proposed
```
Comment (amymok) [inline docs/adr/006-start-off-with-server-side-rendering.md:7]: This should be Accepted before we merge.

**Review by amymok — CHANGES_REQUESTED** [review]
LGTM, we should "Accept" this ADR.

File: docs/adr/006-start-off-with-server-side-rendering.md (hunk 1 above)
Comment (alexsoble) [reply docs/adr/006-start-off-with-server-side-rendering.md]: Good catch!

**Pull request merged**